### ISSUE TITLE
pod =cute

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#8737]
Migrated from [rt.perl.org#41234](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=41234) (status was 'resolved')

Searchable as RT41234$

**Comment by p5pRT** [comment]
### From ruud.vantol@booking.com

#### Created by ruud.vantol+perlbug@booking.com



perl \-Mstrict \-Mwarnings \-le'
=for foo
&nbsp;  bar
=cute
&nbsp;  my $foo;
=cut
&nbsp;  print $foo;
'
doesn't produce an error\. Change "=cute" to something that doesn't start with
"=cut" \(like "=Xcut"\) and it does\.

<details><summary>Perl Info</summary>

```

Flags:
    category=core
    severity=low

Site configuration information for perl v5.8.1:

Configured by build at Tue Oct 21 23:56:27 CEST 2003.

<snipped>

```
</details>

**Comment by p5pRT** [comment]
### From @rgs

Thanks\, for the report\. I've fixed that as change \#29761 in bleadperl\.

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
@rgs - Status changed from 'open' to 'resolved'

**Comment by p5pRT** [comment]
### From @druud62

"Rafael Garcia\-Suarez via RT" schreef&#8203;:

> Thanks\, for the report\. I've fixed that as change \#29761 in bleadperl\.

Thanks\!

\-\- 
Affijn\, Ruud

"Gewoon is een tijger\."
